roles/endpoints/defaults/main.yml

```diff
@@ -3,6 +3,8 @@ endpoints:
   main: "{{ fqdn }}"
   db: "{{ undercloud_floating_ip }}"
   rabbit: "{{ undercloud_floating_ip }}"
+  mongodb: "{{ undercloud_floating_ip }}"
+  magnum: "{{ fqdn }}"
```

Comment:
I think they were needed because of dict lookups failing elsewhere ... not because they were actually being used.